Dockerfile

```diff
@@ -1,19 +1,26 @@
-FROM keboola/base-php56
-MAINTAINER Miro Cillik <miro@keboola.com>
+FROM php:7-cli
```

Comment:
Nemělo by tady být 7.2? Takhle to bere vždycky latest verzi PHP a může nám to neplánovaně upgradnonut na 7.3 nebo třeba 7.9, ne?

Reply:
Zrovna tohle je library tak tady to imho nevadí. Ideální by asi bylo v případě library testovat všechny podporované verze...